.github/workflows/ci.yml

```diff
@@ -188,15 +200,13 @@ jobs:
           echo "Unzip complete"
       - name: Set Environment Variables
         id: set-env
-        shell: bash
+        shell: cmd
         run: |
-          echo "C:\msys64\usr\bin;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\bin;" >> $GITHUB_PATH
-          # Disable SPMD testing
-          echo "NO_DIST=1" >> "$GITHUB_ENV"
-          # Determine dpctl support
-          DPCFLAG="0"
-          if [ $(echo ${{ matrix.PYTHON_VERSION }} | grep '${{ env.DPCTL_PY_VERSIONS }}') ]; then DPCFLAG=''; fi
-          echo "DPCFLAG=${DPCFLAG}" >> "$GITHUB_OUTPUT"
+          echo C:\msys64\usr\bin;>> %GITHUB_PATH%
+          echo NO_DIST=YES>> %GITHUB_ENV%
+          set DPCTL_TEMP="${{ env.DPCTL_PY_VERSIONS }}"
```

Comment:
If this is for windows, shouldn't the quote start before the name of the variable that you are setting?

Reply:
Good catch, luckily in this case the quotation marks do not impact the operation of the underlying check. Will swap it nevertheless

Reply:
Turns out changing this interprets the ```|``` into an operation and will fail. Reverted to original.